i18n/i18n.go

```diff
@@ -69,9 +69,15 @@ import (
 // If translationID is a non-plural form, then the first variadic argument may be a map[string]interface{}
 // or struct that contains template data.
 //
-// If translationID is a plural form, then the first variadic argument must be an integer type
+// If translationID is a plural form, the function accepts two parameter signatures
+// - T(count int, data struct{})
```

Comment:
I think this would be slightly more readable as a numbered list (i.e. `1.` here instead of `-`).